### ISSUE TITLE
pytorch version

### ISSUE THREAD

**Issue by ashutoshpndy** [jialuli-luka/VLN-SIG#7]
Please update the contents of the requirement.txt file because it automatically installs PyTorch version 2.0.1. When we run the pre-training process, we encounter an issue where NaN values appear in the SAR  and other pre-training(tasks) loss calculations.

can you please update the specific versions in requirement.txt you have been using for implementing vlnsig pretraining.

**Comment by jialuli-luka** [comment]
Hi,
    We use Pytorch 1.7.1. You could install it with pip install torch==1.7.1+cu110 torchvision==0.8.2+cu110 torchaudio==0.7.2 -f https://download.pytorch.org/whl/torch_stable.html

**Comment by ashutoshpndy** [comment]
Thanks for your response.
